finrl/meta/paper_trading/paper_trader.py

```
@@ -46,69 +46,65 @@ def __init__(
-            elif drl_lib == "stable_baselines3":
-                from stable_baselines3 import PPO
-
-                try:
-                    # load agent
-                    self.model = PPO.load(cwd)
-                    self.logger.info(f"Successfully load model {cwd}")
-                except:
-                    raise ValueError("Fail to load agent!")
+                self.agent = trainer
+                self.logger.info(f"Restoring from checkpoint path {cwd}")
+            except:
```

Ruff

Do not use bare `except`

(E722)